### DISCUSSION TITLE
I need to login to a website using a digital certificate

### DISCUSSION THREAD

**Discussion by jvitormelo** [seleniumbase/SeleniumBase#2960] [Q&A] [answered]
I have a .p12 file and the password, how can I install it in selenium base?

**Comment by mdmintz** [comment, marked as answer]
The closest I've seen is a way with Python `requests`: https://gist.github.com/erikbern/756b1d8df2d1487497d29b90e81f8068

However, as of today, there doesn't appear to be a Selenium option for it. The StackOverflow post from 4 years ago remains unanswered: https://stackoverflow.com/questions/64116470/selenium-webdriver-chromedriver-is-it-possible-to-provide-p12-certificate-as-c
